arch/arm/include/inttypes.h

```diff
@@ -29,91 +29,138 @@
  * Pre-processor Prototypes
  ****************************************************************************/
 
+#define int         +2
+#define long        +4
```

Comment:
Why this is needed?

Reply:
It's a clever hack I guess from newlib:
https://github.com/bminor/newlib/blob/master/newlib/libc/include/sys/_intsup.h#L57-L58
The compiler may define `__INTPTR_TYPE__`  to 'int', `long` or `'long int`, how can we know which one in the compiler time?
We can define `int` to +2 and `long` to +4, so `__INTPTR_TYPE__` will expand to `+2` for `int` or `+4` or `+2 + 4` for `long int` and `long`.